src/app/shared/utilities/time.ts

```diff
@@ -0,0 +1,38 @@
+const SEC_MILLISEC = 1000
+const HOUR_MIN = 60
+const MIN_SEC = 60
+const DAY_HOUR = 24
+const YEAR_DAY = 365
+const WEEK_DAY = 7
+const MONTH_DAY = 31
```

Comment:
Would be good to have these as enums ?

Reply:
Also be fine to name them with `TO` in between for more readability.

Reply:
Made the changes.